frontend/rolecall/src/app/api/cast_api.service.ts

```diff
@@ -35,6 +35,11 @@ type AllRawCastsResponse = {
   warnings: string[]
 }
 
+type OneRawCastsResponse = {
```

Comment:
Please a comment about what this means, or could we pick a better name? Not clear what "one" means here

Reply:
Done.

Reply:
This line is still missing a comment